plugin.video.mlbtv/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.mlbtv" name="MLB.TV®" version="2022.4.19" provider-name="eracknaphobia">
+<addon id="plugin.video.mlbtv" name="MLB.TV®" version="2022.5.6" provider-name="eracknaphobia">
```

Comment:
this is not compatible with the version available in matrix (and nexus) @eracknaphobia. Version needs to be 2022.5.5 to be lower than 2022.5.5+matrix.1